share/functions/fish_clipboard_paste.fish

```diff
@@ -8,6 +8,8 @@ function fish_clipboard_paste
         set data (xsel --clipboard 2>/dev/null)
     else if type -q xclip
         set data (xclip -selection clipboard -o 2>/dev/null)
+    else if type -q powershell.exe
+        set data (powershell.exe Get-Clipboard | tr -d '\r' 2>/dev/null)
```

Comment:
That `tr` should be `string trim -r -c \r` (no need to silence stderr)

Reply:
Oh nice, thanks for pointing that out. Amended the commit